services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -181,7 +181,11 @@ async def _copy() -> None:
             product_name=product_name,
         ):
             task_progress.update(
-                message=async_job_composed_result.status.progress.composed_message,
+                message=(
+                    async_job_composed_result.status.progress.message.description
```

Comment:
MINOR: a message's description? :-)   I wonder if a `message.text` would be a better name

Reply:
well. this is a hack anyway.